scripts/docker/buildpack/Dockerfile.ubuntu2004

```diff
@@ -58,5 +59,7 @@ RUN set -ex; \
 
 FROM base
 COPY --from=libraries /usr/lib /usr/lib
+RUN true
```

Comment:
Workaround: See https://github.community/t/attempting-to-build-docker-image-with-copy-from-on-actions/16715

Reply:
This by the way happens for me locally as well - without these ``RUN true``s I always have to run the docker build twice over locally before it works... not sure why this happens and if there's a nicer way to avoid it...